nbproject/platform.properties

```diff
@@ -1,7 +1,8 @@
 branding.token=autopsy
+nbjdk.active=default
 # Version of platform that is automatically downloaded
 # IMPORTANT: autopsy/build.xml has a netbeans-plat-version property that MUST be kept in sync (manually)
-netbeans-plat-version=11.3
+netbeans-plat-version=15
```

Comment:
@gdicristofaro is this the Netbeans platform? I'm on platform 12.4. Are you on 15?